src/graph.jl

```diff
@@ -82,7 +82,7 @@ visited: (optional, None) If given, a set of nodes that have
 
 NOTE: If node doesn't exist, the method will return False.
 """
-function ancestor_of(graph::ArbiterGraph, name::Symbol, ancestor::Symbol, visited::NodeSet)
+function ancestor_of(graph::ArbiterGraph, name::Symbol, ancestor::Symbol, visited::NodeSet=NodeSet())
```

Comment:
I'll revert that change. When I said it was reasonable I thought that it wasn't being modified within the function.